node/repo/importmgr/mgr.go

```diff
@@ -98,14 +116,50 @@ func (m *Mgr) Info(id multistore.StoreID) (*StoreMeta, error) {
 	return &sm, nil
 }
 
-func (m *Mgr) Remove(id multistore.StoreID) error {
-	if err := m.mds.Delete(id); err != nil {
-		return xerrors.Errorf("removing import: %w", err)
-	}
-
+func (m *Mgr) Remove(id uint64) error {
 	if err := m.ds.Delete(datastore.NewKey(fmt.Sprintf("%d", id))); err != nil {
 		return xerrors.Errorf("removing import metadata: %w", err)
 	}
 
 	return nil
 }
+
+func (m *Mgr) CARV2FilePathFor(dagRoot cid.Cid) (string, error) {
+	importIDs, err := m.List()
+	if err != nil {
+		return "", xerrors.Errorf("failed to fetch import IDs: %w", err)
+	}
+
+	for _, importID := range importIDs {
+		info, err := m.Info(importID)
+		if err != nil {
+			continue
+		}
+		if info.Labels[LRootCid] == "" {
+			continue
+		}
+		c, err := cid.Parse(info.Labels[LRootCid])
+		if err != nil {
+			continue
+		}
+		if c.Equals(dagRoot) {
+			return info.Labels[LCARv2FilePath], nil
+		}
+	}
+
+	return "", nil
+}
+
+func (m *Mgr) NewTempFile(id uint64) (string, error) {
+	file, err := ioutil.TempFile(m.repoPath, fmt.Sprintf("%d", id))
```

Comment:
`ioutil.TempFile` already appends a random number at the end, I'd just do
```suggestion
func (m *Mgr) NewTempFile() (string, error) {
	file, err := ioutil.TempFile(m.repoPath, fmt.Sprintf("import-temp-", id))
```

Reply:
This should also be a subdirectory in the repo, so users can mount it to a separate volume easily

Reply:
@magik6k Should we allow users to pass in their subdir path for this via the Config ?

Reply:
It can just be a hardcoded directory in the repo - users will be able to link/mount it to different volumes